src/border_agent/border_agent.cpp

```diff
@@ -76,6 +77,7 @@ namespace otbr {
 static const char    kBorderAgentServiceType[]      = "_meshcop._udp";   ///< Border agent service type of mDNS
 static const char    kBorderAgentEpskcServiceType[] = "_meshcop-e._udp"; ///< Border agent ePSKc service
 static constexpr int kBorderAgentServiceDummyPort   = 49152;
+static constexpr int kEpskcRandomGenLen             = 8;
```

Comment:
```suggestion
static constexpr uint8_t kEpskcRandomGenLen             = 8;
```

Reply:
Could you elaborate this suggestion? I personally prefer `int` in general cases.